bilby/gw/detector/calibration.py

```diff
@@ -27,6 +33,9 @@ def read_calibration_file(filename, frequency_array, number_of_response_curves,
     starting_index: int
         Index of the first curve to use within the array. This allows for segmenting the calibration curve array
         into smaller pieces.
+    correction: str
+        How the correction is defined, either to the data (default) or
```

Comment:
I'd suggest either (i) we make `correction='data'` instead of `None`, or (ii) change the text here to say something like `traditional` or `usual` rather than `default`. I suspect (ii) is better, as we want to avoid cases where people don't know which one they are doing.